app/src/main/java/com/hedvig/app/datastore/SeenCrossSellsPreferencesDataMigration.kt

```diff
@@ -0,0 +1,45 @@
+package com.hedvig.app.datastore
+
+import androidx.datastore.core.DataMigration
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.stringSetPreferencesKey
+import com.hedvig.android.owldroid.type.TypeOfContract
+import com.hedvig.app.service.badge.NotificationBadge
+
+class SeenCrossSellsPreferencesDataMigration : DataMigration<Preferences> {
```

Comment:
Given that we're not live with the tab notification yet, we can probably omit the migration 🙂